core-icon.html

```diff
@@ -167,7 +167,11 @@
         }
 
         this.iconset = this.computeIconset(this.icon);
-        this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        if (this.iconsetElement) {
+          this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        } else {
+          console.warn('iconset "' + this.iconset + '" not found!');
```

Comment:
> `console.warn` is not a standard and should likely not go in a production element.

According to MDN `console.log` isn't standard either. I'm not sure where that leaves us.

I agree that the exception is not helpful. However, this seems like an "exceptional" condition. An exception with the appropriate messaging is probably the best way forward.

I ran into this problem again today. Let's update this PR and merge in the improved behavior!